src/lib/internal/helpers/typeahead.ts

```diff
@@ -26,6 +26,13 @@ export type HandleTypeaheadSearch = {
 	 */
 };
 
+/**
+ * Keys to ignore for typeahead so we aren't matching things
+ * like `Shift menu item` or `Control center` or `Alt menu` when
+ * a user presses those keys.
+ */
+const ignoredKeys = new Set(['Shift', 'Control', 'Alt', 'Meta', 'CapsLock', 'NumLock']);
```

Comment:
I think it's better to have a allowList rather than a ignoreList, no? Even better, we can just regex and check if the key is a letter.

Reply:
It can also be symbols though right?

Reply:
Yeah, playing around with native select, you're right.